docs/search/search_for_content.md

```diff
@@ -26,6 +26,11 @@ By selecting an option from a drop-down menu in the top right corner, you can so
 
 ![Sorting menu](img/sorting_menu.png)
 
+The search engine not only displays suggestions, but also supports spell checking.
+If there is a typo in your search phrase, the search engine suggests a related, accurate term.
```

Comment:
```suggestion
If there is a typo in your search phrase, the search engine suggests a related, more accurate term.
```

Reply:
_"If you misspell a search phrase..."_ - so it cannot be correct/accurate. Search engine will then show accurate term, so I'd keep like it is.

Reply:
Ok, I have to admit I misunderstood the sentence and thought you're talking about the results and their accuracy (not the search suggestion). Ok for me!